content/spin/kv-store.md

```diff
@@ -186,13 +186,13 @@ In this section, we use the Spin SDK to open and persist our application's data
 
 ### The Spin SDK Version
 
-If you have an existing application and want to try out this new key/value feature (i.e. a previous application made using the `http-rust` template). You might need to update your existing application's `Cargo.toml` to point to v0.9.0 of the Spin SDK, as shown below:
+If you have an existing application and want to try out this new key/value feature (i.e. a previous application made using the `http-rust` template). You might need to update your existing application's `Cargo.toml` to point to v0.10.0 of the Spin SDK, as shown below:
```

Comment:
This paragraph now becomes a bit fiddly, because if they're on 0.9 they _don't_ need to upgrade.  Maybe: If you have an existing app ... check the Spin SDK reference in Cargo.toml.  If it refers to version 0.8 or earlier, update it to match your upgraded version of Spin.  Or something like that?

I'd also consider dropping "new" when mentioning the K/V feature, but maybe we're not at that point yet?

Reply:
Again, great point, changed in new commit.